uber/_version.py

```diff
@@ -1 +1 @@
-__version__ = '14.0'
+__version__ = '2016.10'
```

Comment:
Seems reasonable.  In case it wasn't clear ``14.0`` referred to MAGFest 14 :)

A date-based version number is probably better.